testdata/scripts/generated.txt

```diff
@@ -0,0 +1,36 @@
+gofumpt -w foo.go
```

Comment:
please also test that `gofumpt -w .` does not modify the file. that is, we don't modify it when it's explicitly an argument, nor when it's implicitly walked.